test/test_parse.c

```diff
@@ -3718,6 +3718,17 @@ START_TEST(test_vhdl2008)
 }
 END_TEST
 
+START_TEST(test_vhdl2019) {
```

Comment:
```suggestion
START_TEST(test_vhdl2019)
{
```

Opening brace goes on new lines for these test macros as well as regular functions.